dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -215,6 +215,16 @@ img.video_thumbnail {
   }
 }
 
+.sign_in.show_only_on_mobile {
+  float: right;
+  margin-right: 10px;
+  margin-bottom: 10px;
+
+  @media screen and (max-device-height: 900px) and (max-device-width: 500px) {
```

Comment:
do we use max-device-height anywhere else?  i thought our mobile layout was keyed entirely off of width?